### PR TITLE
Cell Class: Iteration 1 - Part 1 of 3

### PR DESCRIPTION
**Pull request by Coralruschak**
Created Cell class

initialized cell object with attributes
implemented #empty? and #place_ship methods
created comprehensive tests, all tests passing
perhaps there are still edge cases to review?
any necessary changes can be approached as we go or through an official refactor

### CODE REVIEW

File: spec/cell_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      expect(@cell).to be_a(Cell)
+    end
+
+    it 'has a coordinate' do
+      expect(@cell.coordinate).to eq("B4")
+    end
+  end
+  
+  describe '#ship' do
+    it 'has defaults to nil' do
```
Comment (willfox0409) [inline spec/cell_spec.rb:21]: little typo in the title here I can fix in my next push!

**Comment by willfox0409** [comment]
This is great @Coralruschak, I'm going to merge and work on the next portion of the interaction pattern! Thank you!

File: spec/cell_spec.rb (hunk 1 above)
Comment (Coralruschak) [reply spec/cell_spec.rb:21]: That sounds great and good catch, thanks!
